packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1931,6 +1931,15 @@ test('manager notifiers, with snapshot', async t => {
     totalCollateral: { value: totalCollateral },
   });
 
+  // XXX fudgy numbers
+  totalCollateral += 2n;
+  totalDebt += 429n;
```

Comment:
These  are suspicious. If the point is to get the accounting right, we need to clarify.

Reply:
Good call. Hunted down where the values come from and force pushed the small changes to these lines.